### PR TITLE
Fix onboarding with 0 found integrations

### PR DESCRIPTION
**Pull request by wendevlin**
## Proposed change

If at the Home Assistant onboarding process 0 (additional to the standard ones like: sun, radio browser,...) integrations are found, show an "All set" screen instead of the integrations screen.

image

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

## Summary by CodeRabbit

- **New Features**
	- Introduced celebratory messaging for users completing the onboarding process without integrations, enhancing user feedback.
	- Added new translation strings to improve the interface for device integration, making it more welcoming.

- **Style**
	- Implemented a new CSS class for styling the celebratory icon in the onboarding component.

### CODE REVIEW

File: src/onboarding/onboarding-integrations.ts [hunk 1]
```diff
@@ -117,6 +117,30 @@ class OnboardingIntegrations extends SubscribeMixin(LitElement) {
 
     const foundIntegrations = domains.length;
 
+    // there is a possibility that the user has no integrations
+    if (domains.length === 0) {
```
Comment (piitaya) [inline src/onboarding/onboarding-integrations.ts:121]:
```suggestion
    if (foundIntegrations === 0) {
```

File: src/onboarding/onboarding-integrations.ts [hunk 2]
```diff
@@ -117,6 +117,30 @@ class OnboardingIntegrations extends SubscribeMixin(LitElement) {
...
+            "ui.panel.page-onboarding.integration.all_set"
+          )}
+        </h1>
+        <p>
+          ${this.onboardingLocalize(
+            "ui.panel.page-onboarding.integration.lets_start"
+          )}
+        </p>
+        <div class="footer">
+          <mwc-button unelevated @click=${this._finish}>
```
Comment (piitaya) [inline src/onboarding/onboarding-integrations.ts:135]:
```suggestion
          <ha-button unelevated @click=${this._finish}>
```

File: src/onboarding/onboarding-integrations.ts [hunk 3]
```diff
@@ -117,6 +117,30 @@ class OnboardingIntegrations extends SubscribeMixin(LitElement) {
...
+            ${this.onboardingLocalize(
+              "ui.panel.page-onboarding.integration.finish"
+            )}
+          </mwc-button>
```
Comment (piitaya) [inline src/onboarding/onboarding-integrations.ts:139]:
```suggestion
          </ha-button>
```

File: src/onboarding/onboarding-integrations.ts (hunk 2 above)
Comment (piitaya) [reply src/onboarding/onboarding-integrations.ts:135]: We can convert all mwc-button to ha-button 🙂

**Review by piitaya — COMMENTED** [review]
Nice 🎉
I only added small suggestions.

**Comment by wendevlin** [comment]
@piitaya I added your suggestions.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**